configcatclient/evaluationdetails.py

```diff
@@ -7,17 +7,35 @@ def __init__(self,
                  user=None,
                  is_default_value=False,
                  error=None,
-                 matched_evaluation_rule=None,
-                 matched_evaluation_percentage_rule=None):
+                 matched_targeting_rule=None,
+                 matched_percentage_rule=None):
```

Comment:
According to the [spec](https://docs.google.com/document/d/1-Vi0YnX57P3gW1-_K_kaR1uZECQowajGJtbGwYvb3a8/edit#heading=h.v2bvmvinkak0) this property should be named `matched_percentage_option`.